packages/js/src/ai-assessment-fixes/components/ai-assessment-fixes-button.js

```diff
@@ -25,11 +26,17 @@ import { LockClosedIcon } from "@heroicons/react/solid";
 const AIAssessmentFixesButton = ( { id, isPremium } ) => {
 	const aiFixesId = id + "AIFixes";
 	const [ isModalOpen, , , setIsModalOpenTrue, setIsModalOpenFalse ] = useToggleState( false );
-	const { activeMarker, editorMode, activeAIButtonId } = useSelect( ( select ) => ( {
+	const { activeMarker, activeAIButtonId, editorType } = useSelect( ( select ) => ( {
 		activeMarker: select( "yoast-seo/editor" ).getActiveMarker(),
-		editorMode: select( "core/edit-post" ).getEditorMode(),
 		activeAIButtonId: select( "yoast-seo/editor" ).getActiveAIFixesButton(),
+		editorType: select( "yoast-seo/editor" ).getEditorType(),
```

Comment:
I did not know we had this 😅

Is it possible to remove editor-specific code from this component?

Reply:
On second thought, this is not ideal but I can live with it as it's part of `packages/js`.